modular_skyrat/modules/science_tools/tool_designs.dm

```diff
@@ -0,0 +1,69 @@
+/datum/design/jawsoflife/science
+	id = SCIENCE_JAWS_OF_LIFE_DESIGN_ID // added one more requirment since the Jaws of Life are a bit OP
+	build_path = /obj/item/crowbar/power/science
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
+
+/datum/design/jawsoflife/science/New()
+	name = ("Science " + name)
+	desc += " but lacking the door-opening power"
+
+	return ..()
+
+/datum/design/handdrill/science
+	id = SCIENCE_DRILL_DESIGN_ID
+	build_type = PROTOLATHE | AWAY_LATHE
+	build_path = /obj/item/screwdriver/power/science
+	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
+
+/datum/design/handdrill/science/New()
+	name = ("Science " + name)
+	desc += " with a science paintjob"
+
+	return ..()
+
+/datum/design/laserscalpel
+	construction_time = 1 SECONDS
+
+/datum/design/mechanicalpinches
+	construction_time = 1 SECONDS
+
+/datum/design/searingtool
+	construction_time = 1 SECONDS
+
+/datum/design/healthanalyzer
+	construction_time = 5 SECONDS
+
+/datum/design/healthanalyzer_advanced
+	construction_time = 5 SECONDS
```

Comment:
Why are you adding custom construction timers for these?

Reply:
Sneaking in without a changelog.

Reply:
Because they require one to be used in the exofab, as documented in machines.dm: //Remember, objects utilising this flag should have construction_time and construction_cost vars. Note contruction_cost doesnt exist anymore